README.md

```diff
@@ -80,15 +80,34 @@ bindsym XF86AudioMute exec swayosd-client --output-volume mute-toggle
 # Source volume toggle mute
 bindsym XF86AudioMicMute exec swayosd-client --input-volume mute-toggle
 
+# Volume raise with custom value
+bindsym XF86AudioRaiseVolume + <key> exec swayosd-client --output-volume 15
```

Comment:
These aren't functional without modification so removing the ` + <key>` :)

Reply:
Removed the + <key> from all the examples.